src/Exceptionless.Core/Settings.cs

```diff
@@ -65,11 +65,7 @@ public class Settings {
 
         public string ApplicationInsightsKey { get; private set; }
 
-        public string MetricsServerName { get; private set; }
-
-        public int MetricsServerPort { get; private set; }
-
-        public bool EnableMetricsReporting { get; private set; }
+        public IMetricsConnectionString MetricsConnectionString { get; private set; }
```

Comment:
I think this should just be a string type and we need to add the enable metrics reporting back @ejsmith unless you think this should be part of the connection string. This allowed us to add a setting to disable it without touching the connection string and also allowed easy way to see if we enabled it.